### DISCUSSION TITLE
Inability to generate ping/notification/highlight on raid system message

### DISCUSSION THREAD

**Discussion by ConnorKrammer** [Chatterino/chatterino2#5725] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [x] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Currently, it does not seem to be possible to receive notification of a raid in Chatterino. This would be useful for moderation purposes (e.g. as a signal that a channel needs immediate attention even if you're not looking at it—especially useful for small and usually quiet channels).

The highlights feature would typically allow one to generate an alert sound in response to raid messages (though unfortunately with no ability to discriminate between channels) based on the message pattern Twitch uses, except that system messages appear to be excluded from being highlighted. That's reasonable in the general case, because most users do not in fact want that behaviour.

However, Chatterino *is* able to select system messages with its filter feature, and then match on their text content to determine that they are raid messages. (Twitch also uses the IRC message tags extension to sets the tag `msg-id=raid`, which would be more precise, but I don't believe this is surfaced by Chatterino. If it is, it's not documented on the filters page in the wiki.) This almost achieves the desired effect, and is even more discriminating because filters are based on a single channel only, but there is no way to generate a highlight/notification/sound for each message in a filter pane, so this also does not work.

Overall, it seems like all the requisite pieces are almost in place, but there's no way for a user to actually combine them for the described purpose, which I believe would be very useful (though primarily for mods).

### OS and Chatterino Version

Chatterino 2.4.0-beta (commit 82797898 modified) built with Qt 5.15.2, Windows SDK, MSVC 193331630 Running on Windows 10 Version 2009, kernel: 10.0.19044

**Comment by Felanbird** [comment]
Currently Chatterino believes raid messages are just sub messages, so it is possible as the ability to enable pings for sub messages does exist, but this would also enable sounds for subscriptions, so it's not ideal.

My suggestion would have been to use the filter `flags.sub_message && message.content contains "raiders"` on a duplicate tab, and just mute the normal tab, but the `mute highlight sound` functionality does not respect duplicate tabs, and just mutes all of them. 🧍‍♂️ 

cc: https://github.com/Chatterino/chatterino2/issues/4231#issuecomment-1349237204
